src/main/java/rest/SearchResource.java

```diff
@@ -0,0 +1,332 @@
+package rest;
+
+import com.google.gson.JsonObject;
+import dto.HobbyDTO_IN;
+import dto.HobbyDTO_OUT;
+import dto.MovieInfo;
+import dto.PersonDTO_IN;
+import dto.PersonDTO_OUT;
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.info.Contact;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.servers.Server;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.ArrayList;
+import java.util.List;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@OpenAPIDefinition(
+        info = @Info(
+                title = "CA2 API",
+                version = "1.0",
+                description = "API related to Cphbusiness 3rd semester CS assignment 'CA2'.",
+                contact = @Contact(name = "Github Contributors", url = "https://github.com/MalteMagnussen/CA2/")
+        ),
+        tags = {
+            @Tag(name = "General", description = "API related to CA2"),
+            @Tag(name = "Persons", description = "CRUD-operations for Person"),
+            @Tag(name = "Hobbies", description = "CRUD-operations for Hobby"),
+            @Tag(name = "Movies", description = "Deprecated")
```

Comment:
Nice. Eventually we'll remove it. Just wanted it in here in the beginning as an example and a jump-off point.